x-pack/test/accessibility/apps/painless_lab.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'security']);
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+  const a11y = getService('a11y');
+
+  describe('Accessibility Painless Lab Editor', () => {
+    before(async () => {
+      await PageObjects.common.navigateToApp('painlessLab');
+      await a11y.testAppSnapshot();
```

Comment:
Let's not leave assertions in the `before` of a test file

Move this into it's own `it(...)` block

Reply:
pulled out the assertion from the before block.

Reply:
But we still need that assertion.

Need another `it('renders the page without a11y errors'...` block to handle this case.

Reply:
added